test/katex-spec.js

```diff
@@ -218,24 +218,21 @@ describe("A subscript and superscript parser", function() {
     });
 
     it("should produce the same thing regardless of order", function() {
-        const parseA = stripPositions(getParsed("x^2_3"));
-        const parseB = stripPositions(getParsed("x_3^2"));
-
-        expect(parseA).toEqual(parseB);
+        expect("x^2_3").toParseLike("x_3^2");
```

Comment:
This is only non-trivial change of test cases. Other changes are just `toNotXXX` to `not.toXXX`.